### PR TITLE
Use matplotlib-base as dependency

### PR DESCRIPTION
**Pull request by jni**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] ~~Reset the build number to `0` (if the version changed)~~
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jni** [comment]
@conda-forge-admin please rerender

**Comment by ocefpaf** [comment]
This line https://github.com/conda-forge/matplotlib-feedstock/blob/master/recipe/meta.yaml#L29 (`numpy >=1.10`) is the problem here. I'll try to fix that in `matplotlib` soon unless someone beats me to it. (I'm away today.)

File: recipe/meta.yaml [hunk 1]
```diff
@@ -44,7 +44,7 @@ requirements:
     - six >=1.7.3
     - {{ pin_compatible('numpy') }}
     - scipy >=0.17
-    - matplotlib >=2.0.0
+    - matplotlib-base >=3.0.2
```
Comment (hmaarrfk) [inline recipe/meta.yaml]: You should have 
```
- matplotlib >= 2.0.0  # [py2]
-  matplotlib-base >=3.0.2  # [py3]
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -44,7 +44,8 @@ requirements:
     - six >=1.7.3
     - {{ pin_compatible('numpy') }}
     - scipy >=0.17
-    - matplotlib >=2.0.0
+    - matplotlib-base >=3.0.2  # [py3k]
+    - matplotlib      >=2.0.0  # [py2k]
```
Comment (jakirkham) [inline recipe/meta.yaml]: Should we relax this to just require 2 for both?

Comment (hmaarrfk) [reply recipe/meta.yaml]: yes! Man that PR came out of nowhere and from the previous discussions it seemed matplotlib 2.x wasn't going to be split.

**Comment by jakirkham** [comment]
@conda-forge-admin please re-render.

**Pull request merged**